ProjectManager/ProjectManager/View/ListView.swift

```diff
@@ -0,0 +1,161 @@
+//
+//  ListView.swift
+//  ProjectManager
+//
+//  Created by leewonseok on 2023/01/11.
+//
+
+import UIKit
+
+final class ListView: UIView {
+    let viewModel: ListViewModel
+    
+    weak var mainViewDelegate: MainViewDelegate?
+    weak var cellDelgate: CellDelegate?
+    weak var workFormDelegate: WorkFormDelegate?
+    
+    private let stackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.spacing = 10
+        stackView.axis = .vertical
+        stackView.alignment = .fill
+        stackView.distribution = .fill
+        return stackView
+    }()
+    
+    let tableView: UITableView = {
+        let tableView = UITableView()
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        tableView.register(ListCell.self, forCellReuseIdentifier: ListCell.identifier)
+        tableView.backgroundColor = .systemGray5
+        return tableView
+    }()
+    
+    private let lineView: UIView = {
+        let view = UIView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.backgroundColor = .systemGray2
+        return view
+    }()
+    
+    private let categoryStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.isLayoutMarginsRelativeArrangement = true
+        stackView.layoutMargins = UIEdgeInsets(top: 10, left: 10, bottom: .zero, right: 10)
+        stackView.spacing = 5
+        return stackView
+    }()
+    
+    private let categoryLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.font = .preferredFont(forTextStyle: .title2)
+        return label
+    }()
+    
+    private let categoryCountLabel: CircleLabel = {
+        let label = CircleLabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.backgroundColor = .black
+        label.textAlignment = .center
+        label.textColor = .white
+        return label
+    }()
+    
+    private let blankView: UIView = {
+        let view = UIView()
+        return view
+    }()
+    
+    init(viewModel: ListViewModel) {
+        self.viewModel = viewModel
+        super.init(frame: CGRect())
+        configureTableView()
+        configureBind()
+        configureLayout()
+        configureData()
+        backgroundColor = .systemGray5
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    func didChangeWorkList(works: [Work]) {
+        viewModel.workList = works
+    }
+    
+    private func configureData() {
+        categoryLabel.text = viewModel.category.description
```

Comment:
`description`을 직접 사용하지말고 아래와 같이 사용할 수 있습니다.
```suggestion
        categoryLabel.text = "\(viewModel.category)"
```